### DISCUSSION TITLE
Github is pushing my code to a repository that I don't control. How do I change repositories?

### DISCUSSION THREAD

**Discussion by strikeouts27** [community/community#65134] [Programming Help]
### Body

I  was working on my project and I wanted to do a commit and than push my changes to github. When I did so, it pushed it to an incorrect repository. I tried to go into the correct repository for the code and used the upload button. When I did so, the code said that I had multiple repositories and now it will not let me upload my changes.

What did I try to fix the problem and what am I expecting? I deleted all repositories connected to the code as they were not important to me. I than tried to push my code back up into a new repository.

When I did so I got the following output:

(base) andrewstribling@Andrews-MBP news % git add .
(base) andrewstribling@Andrews-MBP news % git commit -m "first push"
[main 12a6b54] first push
 1 file changed, 1 insertion(+), 1 deletion(-)
(base) andrewstribling@Andrews-MBP news % git push
fatal: The current branch main has no upstream branch.
To push the current branch and set the remote as upstream, use

    git push --set-upstream origin main
So I did what the error suggested to do and I ran the command

(base) andrewstribling@Andrews-MBP news % git push --set-upstream origin main

this produced the output:

(base) andrewstribling@Andrews-MBP news % git push --set-upstream origin main
fatal: 'strikeouts27/django_news.git' does not appear to be a git repository
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.

I have tried taking this issue to github forums however the answer did not solve my problem. 

Github Forums Link: https://github.com/orgs/community/discussions/65001

My GitHub Link: https://github.com/strikeouts27/django_news

The thing is I don't see that repository anywhere on my github page. Is there any way to change the settings and start over with a new repository? Or change where it pushes to a new repository?

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by ggdio** [comment]
Hey there,

Please run the following command and check if the resulting URL is the same as your repo URL:

`git remote get-url origin`

If its not the correct URL then run the following command with the correct URL parameter:

`git remote set-url origin YOUR_REPO_URL`
